### PR TITLE
Added Support to export openshift artifacts

### PR DESCRIPTION
**Pull request by surajssd**
Earlier only support for kubernetes was there but now artifacts for openshift are also exported

Fixes issue #6

### CODE REVIEW

**Comment by surajssd** [comment]
@kadel please review it needs to change many things in it before moving forward.

File: openshift2nulecule/cli/main.py [hunk 1]
```diff
@@ -114,45 +117,50 @@ def run(self):
...
-            filepath, nulecule_dir)))
-        anymarkup.serialize_file(artifacts, filepath, format="json")
+        for provider in exported_project:
+            if args.export_images != "none":
+                if args.export_images == "internal":
+                    only_internal = True
+                elif args.export_images == "all":
+                    only_internal = False
+
+                exported_project[provider].pull_images(args.oc_registry_host,
```
Comment (kadel) [inline openshift2nulecule/cli/main.py]: Like this we are pulling same image twice.
We don't need to pull images for every provider.

File: openshift2nulecule/cli/main.py [hunk 2]
```diff
@@ -114,45 +117,50 @@ def run(self):
...
+                                                       oc.get_username(),
+                                                       oc.get_token(),
+                                                       only_internal)
+
+                # if registy-host is not set do not perform push
+                if args.registry_host:
+                    exported_project[provider].push_images(args.registry_host,
```
Comment (kadel) [inline openshift2nulecule/cli/main.py]: same as for pull

**Comment by kadel** [comment]
@surajssd 
Wouldn't be better to make ExportedProject.artifact as dict so it can hold two types of artifacts (for k8s, and oc provider) rather that creating two ExportedProject objects?

Than we can get rid of duplicate pull/push of images. Because images are same for both providers

**Comment by surajssd** [comment]
@kadel working on it.

**Comment by surajssd** [comment]
@kadel tell me what do you feel of this, maybe give it a try.

**Comment by kadel** [comment]
I think this is great as initial version for OpenShift support.
Good work @surajssd :+1:

**Comment by surajssd** [comment]
@kadel thanks.

**Pull request merged**